lfr/netlistgenerator/v2/gen_strategies/dropxstrategy.py

```diff
@@ -22,183 +23,175 @@ def reduce_mapping_options(self) -> None:
 
-                            if (meter_in_pred1 and nonmeter_in_pred2) or (
-                                nonmeter_in_pred1 and meter_in_pred2
+                                else:
+                                    print("no device found, mix")
+                                    cn = self._construction_graph.get_fignode_cn(
+                                        fignode
+                                    )
+
+                                    for cn_part in cn.mapping_options:
+                                        if cn_part.primitive.mint != "MIXER":
+                                            cn.mapping_options.remove(cn_part)
+                                    pass
+
+                            if (fignode.type is InteractionType.MIX) or (
+                                fignode.type is InteractionType.SIEVE
                             ):
-                                # this is a pico injection
-                                cn = self._construction_graph.get_fignode_cn(fignode)
 
-                                # check mapping options
-                                for cn_part in cn.mapping_options:
-                                    if cn_part.primitive.mint != "PICOINJECTOR":
-                                        # remove if its not [ico injection
-                                        cn.mapping_options.remove(cn_part)
+                                if numTrue == 0:
+                                    # this is continuous
+                                    pass  # temp
+                                else:
+                                    # not continuous
+                                    # for cn_part in cn.mapping_options:
+                                    #     if (cn_part.primitive.mint == "MIXER") or (
+                                    #         cn_part.primitive.mint == "SIEVE"
+                                    #     ):
+                                    #         print("deleting ", cn_part.primitive.mint)
 
-                            # Rule 6
-                            elif meter_in_pred1 and meter_in_pred2:
-                                # this is a droplet merging
-                                pass  # temp
+                                    #         cn.mapping_options.remove(cn_part)
 
-                            else:
-                                raise Exception("can't recognize the device")
-
-                        if (fignode.type is InteractionType.MIX) or (
-                            fignode.type is InteractionType.SIEVE
-                        ):
-
-                            if nonmeter_in_pred1 and nonmeter_in_pred2:
-                                # this is continuous
-                                pass  # temp
-                            else:
-                                # not continuous
-                                pass
+                                    print("not continuous")
+                                    pass
```

Comment:
@ryu-bu can you remove the commented-out code and put in a cleanly formatted comment saying what needs to happen ? 

You're printing out a statement saying not continuous but I'm not sure why that would be the case